samples/nrf_profiler/sample.yaml

```diff
@@ -7,11 +7,13 @@ tests:
     integration_platforms:
       - nrf52dk_nrf52832
       - nrf52840dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp_ns
```

Comment:
Commit message for this one seems outdated. Consider updating it.

Reply:
Updated